python/ray/util/spark/cluster_init.py

```diff
@@ -1164,6 +1164,11 @@ def setup_ray_cluster(
         # Because tasks that require CPU or GPU resources are not scheduled to Ray
         # head node, limit the heap memory and object store memory allocation to the
         # head node.
+        if object_store_memory_head_node is not None:
+            raise ValueError(
+                "If you want to set 'object_store_memory_head_node', you should also "
+                "set num_cpus_head_node to >=1."
```

Comment:
why is that?

Reply:
if num_cpus_head_node and num_gpus_head_node is zero, we assuming no task will be scheduled to this node so we limit the head node memory usage to 128MB, is it good ?

Reply:
There can still be zero cpu task or actors.

Reply:
OK.